barrier3d/barrier3d.py

```diff
@@ -779,6 +756,413 @@ def migrate_dunes(
                 self._SCRagg[time_index] + cellular_shoreline_change
             )  # Reset (make more positive), leaving residual
 
+    @staticmethod
+    @jit(nopython=True, error_model='numpy')
+    def route_overwash(BarrierLength,
+                       nn,
+                       MaxUpSlope,
+                       Shrub_ON,
+                       ShrubPercentCover,
+                       Qshrub_max,
+                       DeadPercentCover,
+                       Dmax,
+                       Qs_min,
+                       Kr,
+                       Ki,
+                       mm,
+                       SL,
+                       Cbb_r,
+                       Cbb_i,
+                       Qs_bb_min,
+                       ShrubDomainFemale,
+                       ShrubDomainMale,
+                       ShrubDomainDead,
+                       ShrubDomainAll,
+                       SalineLimit,
+                       width,
+                       TS,
+                       Discharge,
+                       Rin,
+                       Elevation,
+                       SedFluxIn,
+                       SedFluxOut,
+                       ShrubDomainWidth,
+                       DeadDomainWidth,
+                       inundation,
+                       C):
+
```

Comment:
Can you add a single comment that says what this does?

Reply:
Done, see new commit